gui/scripts/syncthing/core/views/directives/aboutModalView.html

```diff
@@ -0,0 +1,82 @@
+<modal id="about" large="yes" close="yes" status="info" title="{{'About' | translate}}">
+  <h1 class="text-center"><img alt="Syncthing" title="Syncthing" src="assets/img/logo-horizontal.svg" style="vertical-align: -16px" height="100" width="366"/><br/><small>{{versionString()}}</small></h1>
+  <hr/>
+
+  <p translate>Copyright &copy; 2015 the following Contributors:</p>
+  <div class="row">
+    <div class="col-md-12">
+      <ul class="list-unstyled three-columns" id="contributor-list">
```

Comment:
This requires a companion change to `authors.go`, the script that generates this list.